Python/bytecodes.c

```diff
@@ -263,13 +274,10 @@ dummy_func(
             ERROR_IF(res == NULL, error);
         }
 
-        // This is a subtle one. It's a super-instruction for
-        // BINARY_OP_ADD_UNICODE followed by STORE_FAST
-        // where the store goes into the left argument.
-        // So the inputs are the same as for all BINARY_OP
-        // specializations, but there is no output.
-        // At the end we just skip over the STORE_FAST.
-        inst(BINARY_OP_INPLACE_ADD_UNICODE, (left, right --)) {
+        // Part 1's output effect is a lie -- it has no result.
+        // Part 2's input effect is equally a lie, and the two lies
+        // cancel each other out.
+        op(_BINARY_OP_INPLACE_ADD_UNICODE_PART_1, (left, right, unused/1 -- unused)) {
```

Comment:
The lies are so that we can add `_BINARY_OP_INPLACE_ADD_UNICODE_PART_1` to the family and define the thing as a proper super-instruction.

Since you react rather strongly to this I think maybe I should just roll that back. See also https://github.com/faster-cpython/ideas/issues/495#issuecomment-1341756507.